src/erlfmt_cli.erl

```diff
@@ -18,6 +18,7 @@
 -record(config, {
     verbose = false :: boolean(),
     require_pragma = false :: boolean(),
+    insert_pragma = false :: boolean(),
```

Comment:
I wonder if it would make sense to also have a single `pragma = ignore :: ignore | require | insert` option here. We could avoid the extra conversion before calling `erlfmt:format_file`. Validation we're not using both option could be done in case we're setting an option when `Pragma =/= ignore`.

Reply:
I think this fixed in https://github.com/WhatsApp/erlfmt/pull/63/commits/7c2cfc138d505033f88ca55b157af3fb2aa80192